DashWallet/Sources/Models/CrowdNode/CrowdNodeConstants.swift

```diff
@@ -30,5 +30,6 @@ enum CrowdNodeConstants {
 
     static var minimumRequiredDash = UInt64(1_000_000)
```

Comment:
i think it's better to write `static var minimumRequiredDash: UInt64 = 1_000_000`, in this case you see the type right away